pbjam/plotting.py

```diff
@@ -15,19 +15,50 @@
 
 import matplotlib.pyplot as plt
 import astropy.convolution as conv
-import pbjam, os, corner, warnings
+import pbjam, os, corner, warnings, logging
 import numpy as np
 from pymc3.gp.util import plot_gp_dist
 import astropy.units as u
 import pymc3 as pm
+import pandas as pd
+from .jar import to_log10
+
+
 
 class plotting():
 
     def __init__(self):
         pass
 
-    def plot_echelle(self, pg=None):
+    def _save_my_fig(self, fig, figtype, path, ID):
+        """ Save the figure object
+        
+        Saves the figure object with a predefined path name pattern.
+
+        Parameters
+        ----------
+        fig : TYPE
+            DESCRIPTION.
+        figtype : TYPE
+            DESCRIPTION.
+        path : TYPE
+            DESCRIPTION.
+        ID : TYPE
+            DESCRIPTION.
+
+        Returns
+        -------
+        None.
+
         """
+        # TODO there should be a check if path is full filepath or just dir
```

Comment:
Did you mean to leave a todo here?

Reply:
Yeah, since we do similar things in other places in PBjam I thought I'd write a more general function that makes this check properly and more consistently.